redaxo/src/addons/metainfo/pages/content.metainfo.php

```diff
@@ -20,7 +20,50 @@
 $panels[] = '<dt>'.rex_i18n::msg('created_on').'</dt><dd>'.rex_formatter::intlDate($article->getValue('createdate')).'</dd>';
 $panels[] = '<dt>'.rex_i18n::msg('updated_by').'</dt><dd>'.rex_escape($article->getValue('updateuser')).'</dd>';
 $panels[] = '<dt>'.rex_i18n::msg('updated_on').'</dt><dd>'.rex_formatter::intlDate($article->getValue('updatedate')).'</dd>';
-$panels[] = '<dt>'.rex_i18n::msg('status').'</dt><dd class="'.$articleStatusTypes[$status][1].'">'.$articleStatusTypes[$status][0].'</dd>';
+
+
+$articleClass = $articleStatusTypes[$status][1];
+$articleStatus  = $articleStatusTypes[$status][0];
+$articleIcon = $articleStatusTypes[$status][2];
+$structureContext = new rex_structure_context([
+    'article_id' => rex_request('article_id', 'int'),
+]);
+
+if (0 == $article->getValue('startarticle')) {
+    if (rex::getUser()->hasPerm('publishArticle[]')) {
```

Comment:
```suggestion
    if (rex::requireUser()->hasPerm('publishArticle[]')) {
```

Das behebt die psalm-Meldung.